sanic/config.py

```diff
@@ -1,8 +1,17 @@
-import os
-import types
+from os import environ as os_environ
 
-from sanic.exceptions import PyFileError
-from sanic.helpers import import_string
+from typing import Union, Any
+
+from .utils import str_to_bool, load_module_from_file_location
+
+# NOTE(tomaszdrozdz): remove in version: 21.3
+# We replace from_envvar(), from_object(), from_pyfile() config object methods
+# with one simpler update_config() method.
+# We also replace "loading module from file code" in from_pyfile()
+# in a favour of load_module_from_file_location().
+# Please see pull request: 1903
+# and issue: 1895
```

Comment:
Is it ok now ? Or should I change something ?

Reply:
:+1: